### PR TITLE
feat(web): new sidebar

### PR DESCRIPTION
**Pull request by McPizza0**
add new (auto collapsing) sidebar based up updated designs

### CODE REVIEW

**Comment by McPizza0** [comment]
* **#469** <a href="https://app.graphite.dev/github/pr/un/inbox/469?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @McPizza0 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/web/src/app/[orgShortCode]/_components/sidebar-content.tsx [hunk 1]
```diff
@@ -0,0 +1,425 @@
...
+      )}>
+      <OrgMenu />
+      <div
+        className={cn(
+          'flex w-full grow flex-col items-start justify-start gap-4 p-0'
+        )}>
+        <div className="flex w-full flex-col gap-0 p-0">
+          <span className="text-slate-10 p-1 text-[10px] font-semibold">
+            SPACES
+          </span>
```
Comment (BlankParticle) [inline apps/web/src/app/[orgShortCode]/_components/sidebar-content.tsx:87]: call it `Spaces` and add `uppercase` class to the span, its better for accessibility

**Review by BlankParticle — CHANGES_REQUESTED** [review]
one nitpick, and the less we do custom hooks, the better

File: apps/web/src/app/[orgShortCode]/_components/sidebar-content.tsx (hunk 1 above)
Comment (McPizza0) [reply apps/web/src/app/[orgShortCode]/_components/sidebar-content.tsx:87]: good point!

**Review by BlankParticle — APPROVED** [review]

**Comment by McPizza0** [comment]
### Merge activity

* **May 25, 5:59 AM EDT**: @McPizza0 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/un/inbox/469).

**Pull request merged**